### PR TITLE
elasticsearch 7.0.0

### PR DESCRIPTION
**Pull request by ogidow**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/elasticsearch.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Elasticsearch < Formula
   desc "Distributed search & analytics engine"
   homepage "https://www.elastic.co/products/elasticsearch"
-  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-6.7.0.tar.gz"
-  sha256 "ffe589cf2e74a44a35f059338fdef30dffcb7196d6bfd916d64173b32c9a2451"
+  url "https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-7.0.0-darwin-x86_64.tar.gz"
```
Comment (SMillerDev) [inline Formula/elasticsearch.rb]: Does this contain precompiled software specific to macOS?

**Comment by fxcoudert** [comment]
Hi, thanks for the pull request! Homebrew formulas need to build from source to be acceptable in the core repository: https://docs.brew.sh/Acceptable-Formulae

**Comment by chrmoritz** [comment]
Also all `beats` which are still at version 6.2.0 won't be compatible with `elasticsearch` 7. With 7.0 we can make `filebeat` oss only compatible, but all other beats (except the already migrated `auditbeat` and `metricbeat`) will be incompatible with `elasticsearch` 7. https://www.elastic.co/de/support/matrix#matrix_compatibility

I think the elastic 7.0 upgrade will get quite complicated here.

**Comment by fxcoudert** [comment]
OK, closing for now until we have a coherent migration plan.

**Pull request closed without merging**